### PR TITLE
CLOUD-387 account management (resource & user)

### PR DESCRIPTION
**Pull request by oleewere**

### CODE REVIEW

**Comment by matyix** [comment]
##In Sultans

1. Add the ability to make user admin
2. If a user is inactivated that change the error message

**Comment by oleewere** [comment]
depends on https://github.com/sequenceiq/cloudbreak/commit/69db3372109bb49c7a0b15b21dd230dda65e5f0a

File: app/static/js/controllers/accountuserController.js [hunk 1]
```diff
@@ -34,15 +37,33 @@ angular.module('uluwatuControllers').controller('accountuserController', ['$scop
...
+            }, function (error) {
+                $scope.modifyStatusMessage(error.data.message);
+                $scope.modifyStatusClass("has-error");
+            })
+        }
+        
+        function getUsersForAccount() {
+            UserPermission.get(function(success){
+               $scope.user.admin = success.admin;
+               if ($scope.$parent.user.admin != undefined && $scope.$parent.user.admin) {
```
Comment (biharitomi) [inline app/static/js/controllers/accountuserController.js]: Here $scope.user.admin should be evaluated and used.

File: app/static/css/Cloudbreak.min.css [hunk 2]
```diff
@@ -5593,3 +5593,7 @@ footer p {
 footer a:hover {
     color: #9bd5e3
 }
+
+.public-account-info {
```
Comment (doktoric) [inline app/static/css/Cloudbreak.min.css]: You shouldn't put here your class

**Pull request merged**